docarray/array/storage/annlite/backend.py

```diff
@@ -25,6 +27,7 @@ class AnnliteConfig:
     ef_construction: Optional[int] = None
     ef_search: Optional[int] = None
     max_connection: Optional[int] = None
+    columns: Optional[List[Tuple[str, str]]] = None
```

Comment:
although there might not be actual type mapping for annlite, let's still add this type map to annlite backend mixin:
```python
TYPE_MAP = {'str': 'str', 'float': 'float', 'int': 'int'}
```
and use `_map_type ` to map configs. At least, in case we opt to change the definition of type mapping, we will be aware of it from docarray instead of sending to downstream annlite